crates/red_knot_python_semantic/resources/mdtest/assignment/unbound.md

```diff
@@ -58,3 +58,16 @@ class C:
 
 reveal_type(C.y)  # revealed: Literal[1] | Literal["abc"]
 ```
+
+## Unbound function local
```

Comment:
This test also feels more suitable to the `scopes/` directory than the `assignment/` directory. But it looks like that's also true of the previous two existing tests. I'd probably move them all to `scopes/unbound.md`, but it's also fine if we don't do that in this PR.